builder/dockerfile/internals.go

```diff
@@ -447,11 +447,11 @@ func (b *Builder) processImageFrom(img builder.Image) error {
 // If no image is found, it returns `(false, nil)`.
 // If there is any error, it returns `(false, err)`.
 func (b *Builder) probeCache() (bool, error) {
```

Comment:
Comment should be updated. Right now it says "probeCache checks if `b.docker` implements builder.ImageCache and image-caching is enabled"